Token.js

```diff
@@ -2174,8 +2174,11 @@ class Token {
 					let playerTokenId = $(`.token[data-id*='${window.PLAYER_ID}']`).attr("data-id");
 					let playerTokenAuraIsLight = (playerTokenId == undefined) ? true : window.TOKEN_OBJECTS[playerTokenId].options.auraislight;
 					if(!window.DM && playerTokenAuraIsLight){
```

Comment:
Yeah I'd thought the same with scale factor but it isn't all tokens or all players. That's why I'm not sure and it might be a remanent of when we were getting NaN in some of those options.

 The other day I had a player where only 1 of his player tokens was doing it - all other tokens worked fine for him including other player tokens. So it might be a combination of the scale_factor and the other options. Redropping that particular token fixed it, which leads me to suspect in that case it was the options.size but not 100%. 

As far as the other stuff - auraislight is just light enabled on the player token. It's the old option name and I didn't want to change it everywhere from where we used to use light auras. 

This would set it to false
![image](https://user-images.githubusercontent.com/65363489/233367449-12106f4b-5cf6-4b3f-9f77-0ca2507f7a74.png)

When vision/light is enabled on the player token it checks for walls and line of sight. If vision/light is disabled it ignores all that. So if it's enabled on the token lets look for walls to collide with.

The other where it's undefined is for if we don't have a token on the map (like a DM viewing it from another character, or tabletop tvs for a room of people) where you still want to move tokens around it looks for walls. In this view where there is no player token all other PCs vision is shared so the tabletop tv/DM view works without having to add another token.

Reply:
> OK, that makes sense, thank you.
> 
> This is unrelated to this PR, but is there a way to turn that off by default? Or maybe it would be better to completely ignore it for scenes that don't have walls on them? It seems like we're doing extra/unnecessary processing on scenes that don't use those features. I don't fully understand the extent of that, though. If it's negligible, then it's probably not worth changing, but any performance gains would be nice. With that in mind, we could probably do something like this in the `start` function to avoid fetching the PC token on every `drag` iteration `var shouldCheckRayCasting = !window.DM && playerTokenAuraIsLight;`

We can certainly drop that in start. You can disable light stuff in the scene and in token customizations. I may need to add the scene option to this check as I don't think it techinically disables it on the token.